src/CLI/clitree/cli-xml/ipv4.xml

```diff
@@ -39,6 +39,83 @@ limitations under the License.
         >
     <ACTION builtin="clish_nop"/>
     </COMMAND>
+    
+<!--====================show ip arp <ip-addr> commands ======================-->
+    <COMMAND
+        help="show ARP commands"
+        name="show ip arp">
+        <PARAM
+            name="arp-subcommands"
+            help="ARP subcommands"
+            mode="switch"
+            ptype="SUBCOMMAND"
+            optional="true">
+                <PARAM
+                    help="ARP entry for this IP address"
+                    name="ip-addr"
+                    ptype="IP_ADDR"
+                    optional="true">
+                </PARAM>
+        </PARAM>
+       <ACTION>
+           if test "${arp-subcommands}" = "ip-addr"; then&#xA;
+               python $SONIC_CLI_ROOT/sonic-cli-if.py get_openconfig_if_ip_interfaces_interface_subinterfaces_subinterface_ipv4_neighbors arp_show.j2 ip ${ip-addr}&#xA;
+           else&#xA;
+               python $SONIC_CLI_ROOT/sonic-cli-if.py get_openconfig_if_ip_interfaces_interface_subinterfaces_subinterface_ipv4_neighbors arp_show.j2 None None&#xA;
+           fi&#xA;
+       </ACTION>
+    </COMMAND>
+<!--====================show ip arp interface <intf> [summary] ============-->
+    <COMMAND
+        help="ARP entries for this interface"
+        name="show ip arp interface">
+        <PARAM
+            name="Ethernet"
+            help="Ethernet IEEE 802.3"
+            ptype="SUBCOMMAND"
+            mode="subcommand">
+            <PARAM
+                name="phy-if-id"
+                help="Physical interface(Multiples of 4)"
+                ptype="UINT">
+                <PARAM
+                    help="Summary of ARP entries"
+                    name="summary"
+		            ptype="SUBCOMMAND"
+		            mode="subcommand"
+                    optional="true">
+                </PARAM>
+            </PARAM>
+        </PARAM>
+        <ACTION>
+            if test "${summary}" = ""; then&#xA;
+                python $SONIC_CLI_ROOT/sonic-cli-if.py get_openconfig_if_ip_interfaces_interface_subinterfaces_subinterface_ipv4_neighbors arp_show.j2 Ethernet${phy-if-id} None&#xA;
```

Comment:
Just for understanding, why do we pass None in the python script calls?

Reply:
Since I'm using same openconfig API for all dumps with different args, it helps to print all args in the actioner.